packages/binding-mqtt/src/mqtt-broker-server.ts

```diff
@@ -115,7 +115,9 @@ export default class MqttBrokerServer implements ProtocolServer {
         }
 
         // connect incoming messages to Thing
-        this.broker.on("message", this.handleMessage);
+        this.broker.on("message", (receivedTopic, rawPayload, packet) =>
+            this.handleMessage(receivedTopic, rawPayload, packet)
+        );
```

Comment:
```suggestion
        this.broker.on("message", this.handleMessage.bind(this));
```

To keep things short I usually use the `bind` function. @danielpeintner what do you prefer?

Reply:
Either way is fine by me 👍